src/AudioAnalysisTools/Events/Types/ChirpEvent.cs

```diff
@@ -0,0 +1,86 @@
+// <copyright file="ChirpEvent.cs" company="QutEcoacoustics">
+// All code in this file and all associated files are the copyright and property of the QUT Ecoacoustics Research Group (formerly MQUTeR, and formerly QUT Bioacoustics Research Group).
+// </copyright>
+
+namespace AudioAnalysisTools
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using Acoustics.Shared;
+    using AudioAnalysisTools.Events;
+    using AudioAnalysisTools.Events.Drawing;
+    using AudioAnalysisTools.Events.Interfaces;
+    using AudioAnalysisTools.Events.Tracks;
+    using SixLabors.ImageSharp.Processing;
+
+    public class ChirpEvent : SpectralEvent, ITracks<Track>
+    {
+        private readonly double maxScore;
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="ChirpEvent"/> class.
+        /// </summary>
+        /// <remarks>
+        /// MaxScore establishes a scale for the chirp score. Typically the amplitude of track points is decibels.
+        /// A satisfactory maxScore is 12.0 decibels, since this is a high SNR in enviornmental recordings.
+        /// The normalised score is a linear conversion from 0 - maxScore to [0, 1].
+        /// </remarks>
+        /// <param name="chirp">A chirp track consisting of a sequence of spectral points.</param>
+        /// <param name="maxScore">A maximum score used to normalise the track score.</param>
+        public ChirpEvent(Track chirp, double maxScore)
+        {
+            this.Tracks.Add(chirp);
+            this.ScoreRange = new Interval<double>(0, maxScore);
```

Comment:
`maxScore` should be an interval.

Note you can use any of the following syntaxes where you need to create an interval:

- `(0, someNumber)`
- `someNumber.AsIntervalFromZero()`
- `0.AsIntervalTo(someNumber)`
- `new Interval<double>(0, someNumber)`

Reply:
This constructor has been changed for all four track type events.